src/zope/schema/tests/test__bootstrapfields.py

```diff
@@ -967,6 +967,7 @@ def test_fromUnicode_hit(self):
     def test_normalization(self):
         deadbeef = unicodedata.normalize('NFD', b'\xc3\x84\xc3\x96\xc3\x9c'.decode('utf-8'))
         txt = self._makeOne()
+        self.assertEqual(txt.unicode_normalization, 'NFC')
```

Comment:
It might be good to have a test for this specific scenario, that things work after a pickle trip through legacy data. Maybe something like this:
```python
def test_normalization_after_pickle(self):
    import pickle
    orig_makeOne = self._makeOne
    def makeOne(**kwargs):
        result = orig_makeOne(**kwargs)
        if not kwargs:
            # We should have no state to preserve
            result.__dict__.clear()

        result = pickle.loads(pickle.dumps(result)
        return result

    self._makeOne = makeOne
    self.test_normalization()
```

Reply:
as you wish! thx for the test!